unittest/test_Config.cpp

```diff
@@ -228,19 +228,39 @@ TEST_CASE("Config::update_from_file, error handling")
     Util::write_file("ccache.conf", "max_files =");
     REQUIRE_THROWS_WITH(
       config.update_from_file("ccache.conf"),
-      Equals("ccache.conf:1: invalid unsigned integer: \"\""));
+      Equals("ccache.conf:1: invalid unsigned integer: (empty string)"));
 
     Util::write_file("ccache.conf", "max_files = -42");
     REQUIRE_THROWS_WITH(
       config.update_from_file("ccache.conf"),
-      Equals("ccache.conf:1: invalid unsigned integer: \"-42\""));
+      Equals("ccache.conf:1: invalid unsigned integer (signed value passed "
+             "where unsigned was expected): \"-42\""));
```

Comment:
I'm sorry, but I think that this error message is worse than the previous one.

Reply:
improved ?!